pkg/server/commands/listusers/list_users_rpc_test.go

```diff
@@ -2,301 +2,1245 @@ package listusers
 
 import (
 	"context"
+	"fmt"
 	"testing"
 
-	parser "github.com/craigpastro/openfga-dsl-parser/v2"
-	"github.com/google/go-cmp/cmp"
 	"github.com/oklog/ulid/v2"
 	openfgav1 "github.com/openfga/api/proto/openfga/v1"
 	"github.com/openfga/openfga/pkg/storage/memory"
+	"github.com/openfga/openfga/pkg/testutils"
 	"github.com/openfga/openfga/pkg/tuple"
 	"github.com/openfga/openfga/pkg/typesystem"
 	"github.com/stretchr/testify/require"
-	"google.golang.org/protobuf/protoadapt"
-	"google.golang.org/protobuf/testing/protocmp"
+	"go.uber.org/goleak"
+	"google.golang.org/protobuf/types/known/structpb"
 )
 
-func TestListUsers(t *testing.T) {
+type ListUsersTests []struct {
+	name                  string
+	TemporarilySkipReason string // Temporarily skip test until functionality is fixed
```

Comment:
why is `TemporarilySkipReason` exported?  (vs. temporarilySkipReason)

Reply:
To stand-out against the other, more permanent struct properties. This will be removed before merging into main so just want to make abundantly sure we don't miss.